components/vars/css/components/spectrum-icon.css

```diff
@@ -42,4 +42,18 @@
   --spectrum-icon-success-small-height: var(--spectrum-global-dimension-size-175);
   --spectrum-icon-triplegripper-height: var(--spectrum-global-dimension-size-85);
   --spectrum-icon-workflow-medium-size: var(--spectrum-alias-workflow-icon-size-m);
+
+  --spectrum-icon-user-defined-height: var(--spectrum-icon-tshirt-size-height);
+  --spectrum-icon-user-defined-width: var(--spectrum-icon-tshirt-size-width);
+  --spectrum-icon-tshirt-size-height: var(--spectrum-icon-default-height);
+  --spectrum-icon-tshirt-size-width: var(--spectrum-icon-default-width);
+  --spectrum-icon-default-height: var(--spectrum-alias-workflow-icon-size-m);
+  --spectrum-icon-default-width: var(--spectrum-alias-workflow-icon-size-m);
+
+  --spectrum-ui-icon-user-defined-height: var(--spectrum-ui-icon-tshirt-size-height);
+  --spectrum-ui-icon-user-defined-width: var(--spectrum-ui-icon-tshirt-size-width);
+  --spectrum-ui-icon-tshirt-size-height: var(--spectrum-ui-icon-default-height);
+  --spectrum-ui-icon-tshirt-size-width: var(--spectrum-ui-icon-default-width);
+  --spectrum-ui-icon-default-height: var(--spectrum-alias-workflow-icon-size-m);
+  --spectrum-ui-icon-default-width: var(--spectrum-alias-workflow-icon-size-m);
```

Comment:
Let's move these into the `components/icon/index.css` file so they don't get blown away when we update DNA.

Reply:
I tried moving them to either `components/icon/index.css` or `components/icon/icons.css`. But none of them work, these variable never show up in `dist/index-vars.css`.

Reply:
Hmm interesting. We should figure another way of shoe-horning these in, then, as this makes DNA updates painful. Any ideas?

Reply:
@lazd,
Looks like these spectrum-css private custom properties could be placed in `components/vars` and should be part of `spectrum-global.css`. Can I create folder named -`components/vars/private` and change the gulp script to concat these properties to `spectrum-global.css`?

Reply:
That sounds like a great idea, though I'm not sure about `private`... Maybe `local` or `custom` or something?